spec/support/webmock_helper.rb

```diff
@@ -1,84 +1,88 @@
-    .with(
-      body: {From: "+15555555555", Body: "Execute Order 66 - https://42ni.short.gy/jzTwdF", To: "+12222222222"},
-      headers: {
-        "Content-Type" => "application/x-www-form-urlencoded",
-        "Authorization" => "Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ=="
-      }
-    )
-    .to_return(body: "{\"error_code\":null, \"status\":\"sent\", \"body\":\"Execute Order 66 - https://42ni.short.gy/jzTwdF\"}")
-  stub_request(:post, "https://api.twilio.com/2010-04-01/Accounts/articuno34/Messages.json")
-    .with(
-      body: {"Body" => "It's been 60 days or more since you've reached out to these members of your youth's network:\n", "From" => "+15555555555", "To" => "+12222222222"},
-      headers: {
-        "Accept" => "application/json",
-        "Accept-Charset" => "utf-8",
-        "Authorization" => "Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==",
-        "Content-Type" => "application/x-www-form-urlencoded"
-      }
-    )
-    .to_return(body: "{\"error_code\":null, \"status\":\"sent\", \"body\":\"It's been 60 days or more since you've reached out to these members of your youth's network:\\n\"}")
-  stub_request(:post, "https://api.twilio.com/2010-04-01/Accounts/articuno34/Messages.json")
-    .with(
-      body: {"Body" => "• test", "From" => "+15555555555", "To" => "+12222222222"},
-      headers: {
-        "Accept" => "application/json",
-        "Accept-Charset" => "utf-8",
-        "Authorization" => "Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==",
-        "Content-Type" => "application/x-www-form-urlencoded"
-      }
-    )
-    .to_return(body: "{\"error_code\":null, \"status\":\"sent\", \"body\":\"• test\"}")
-  stub_request(:post, "https://api.twilio.com/2010-04-01/Accounts/articuno34/Messages.json")
-    .with(
-      body: {"Body" => "If you have made contact with them in the past 60 days, remember to log it: https://42ni.short.gy/jzTwdF", "From" => "+15555555555", "To" => "+12222222222"},
-      headers: {
-        "Accept" => "application/json",
-        "Accept-Charset" => "utf-8",
-        "Authorization" => "Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==",
-        "Content-Type" => "application/x-www-form-urlencoded"
-      }
-    )
-    .to_return(body: "{\"error_code\":null, \"status\":\"sent\", \"body\":\"If you have made contact with them in the past 60 days, remember to log it: https://42ni.short.gy/jzTwdF\"}")
-  stub_request(:post, "https://api.twilio.com/2010-04-01/Accounts//Messages.json")
-    .with(
-      body: {"Body" => /volunteer\sconsole\saccount\shas\sbeen\sreactivated/, "From" => nil, "To" => ""},
-      headers: {
-        "Accept" => "application/json",
-        "Accept-Charset" => "utf-8",
-        "Accept-Encoding" => "gzip;q=1.0,deflate;q=0.6,identity;q=0.3",
-        "Authorization" => "Basic Og==",
-        "Content-Type" => "application/x-www-form-urlencoded",
-        "User-Agent" => "twilio-ruby/5.67.2 (linux x86_64) Ruby/3.1.0"
-      }
-    )
-    .to_return(status: 200, body: "", headers: {})
+  module TwilioAPI
```

Comment:
Can we put these stubbed request modules in their own files for easier maintainability?

Reply:
so instead of modules, you want to put twilio stubs and short io stubs into their own separate files?

Reply:
I believe we can still have them as modules but have a hierarchy as follows for clarity and seperation:
```
-- support
    -- subbed_requests
        -- twilio_api.rb
        -- short_io_api.rb
```
A file such as `twilio_api.rb` would look like this:
```ruby
# twilio_api.rb

module StubbedRequests
    module TwilioAPI
        def self.allowed_sites(blacklist)
             lambda { |uri|
                 blacklist.none? { |site| uri.host.include?(site) }
             }
        end

        # stubs
    end
end
```

Reply:
I like that idea. Very clear.